### ISSUE TITLE
ComboBox in CSSLayout

### ISSUE THREAD

**Issue by ropoman** [vaadin/framework#8975]
Hi,

I noticed some strange behaviour of ComboBox in CssLayout that I cannot explain:

1. When embedded in a HTML page (see #8778) and the page is scrolled the combobox optionlist appears at a wrong position since vaadin 8 in Firefox and IE
2. When not embedded in a HTML page the combobox optionlist is fixed and not scrolled with the page. Only embedded it works
3. When de-selecting Firefox 
[TestDesign1.html.txt](https://github.com/vaadin/framework/files/881312/TestDesign1.html.txt)
the combobox closes unless in CssLayout

I have the problem 1 only when using a design. I tried reproducing it with the code commented out below but there the problem does not show.

```
public class Test extends UI {
	@Override
	protected void init(VaadinRequest vaadinRequest) {
		/*
		VerticalLayout vl = new VerticalLayout();
		for (int i=0; i<10; i++)
			vl.addComponent(new Label(""+i));
		CssLayout cssLayout = new CssLayout();
		ComboBox<String> cb = new ComboBox<>();
		cssLayout.addComponent(cb);
		vl.addComponent(cssLayout);
		VerticalLayout vl2 = new VerticalLayout();
		cb = new ComboBox<>();
		vl2.addComponent(cb);
		vl.addComponent(vl2);
		for (int i=0; i<10; i++)
			vl.addComponent(new Label(""+i));
		setContent(vl);	
		*/
		setContent(new TestDesign());
 	}
}
```

I create a new issue since #8778 is closed, there is no reaction to comments and I don't know how to re-open

**Comment by ropoman** [comment]
I noticed that I used vaadin7-combo-box only once in the design, so the problem is independent of the layout and concerns on the Vaadin7 ComboBox. The solution is to migrate to the v8 ComboBox :)
